### PR TITLE
User can create an entry Malcolm Abbott 

### PR DESCRIPTION
**Pull request by Malcolm-Abbott**
https://github.com/Malcolm-Abbott/code-journal/assets/136769102/6654ec36-2614-4c2f-9869-c258cdc348e5

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,10 +17,49 @@
...
+            </div>
+            <div class="column-full label-box">
+              <label for="notes">Notes</label>
+            </div>
+            <div class="column-full textarea-box">
+              <textarea id="notes" name="notes"></textarea>
+            </div>
+            <div class="column-full">
+              <label for="save-button"></label>
+            </div>
```
Comment (shawnkost) [inline index.html]: This `label` isn't needed. We only need to include `label`s when there's text inside of it associated with an `input` or `textarea`, etc. Like how we have "Title" above the title `input`

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+const $inputTitle = document.querySelector('#title');
+
+function formHandler(event) {
+  event.preventDefault();
+  const title = $inputTitle.value;
+  const photo = $inputPhoto.value;
+  const values = {
+    title,
+    photo,
+  };
```
Comment (shawnkost) [inline js/main.js]: You should also be storing the value from the `textarea` inside of here

File: js/main.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  values.entryId = data.nextEntryId;
+  data.entries.push(values);
```
Comment (shawnkost) [inline js/main.js]: `values` should be getting added to the beginning of the array

**Review by shawnkost — CHANGES_REQUESTED** [review]
Few changes here Malcolm!

File: index.html (hunk 1 above)
Comment (Malcolm-Abbott) [reply index.html]: i just thought it was good practice due to screen readers

File: css/styles.css [hunk 4]
```diff
@@ -92,4 +89,4 @@ img {
   header {
     padding-left: 4rem;
   }
-}
+} */
```
Comment (shawnkost) [inline css/styles.css]: Remove all commented out code!

File: js/main.js [hunk 5]
```diff
@@ -15,14 +15,17 @@ const $inputTitle = document.querySelector('#title');
 
 function formHandler(event) {
   event.preventDefault();
+  const $textArea = document.querySelector('#notes');
```
Comment (shawnkost) [inline js/main.js]: You shouldn't be doing `querySelector`s inside functions. By having it inside of here, you're forcing us to traverse the DOM over and over again every single time this function gets called. If you just do it once globally, we don't need to search again

**Review by shawnkost — CHANGES_REQUESTED** [review]
Just a couple more small tweaks here Malcolm!

File: index.html (hunk 1 above)
Comment (shawnkost) [reply index.html]: It is, but only if there's text inside of it. With no text, there's nothing for the screen reader to read out

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**